.ci_support/linux_ppc64le_.yaml

```diff
@@ -9,7 +9,7 @@ channel_sources:
 channel_targets:
 - conda-forge main
 docker_image:
-- quay.io/condaforge/linux-anvil-ppc64le
+- quay.io/condaforge/linux-anvil-cos7-x86_64
```

Comment:
This does not look correct too

Reply:
Same as above. Let's have any discussion in that thread instead: https://github.com/conda-forge/pigz-feedstock/pull/20#discussion_r1354267539